plugins/proxmox/proxmox_count

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+#%# family=auto
+#%# capabilities=autoconf
+
+# Michael Grote
+# michael.grote ät posteo.de
+# Outputs the running, stopped and total counts of ProxMox VMs and LXCs.
+# needs to be run as user: root AND group: root
+# [proxmox_count]
+# user root
+# group root
+
+# Pfade zu binaries
+wqm=$(command -v qm)
+wpct=$(command -v pct)
+# Berechne VMs
+total_vm=$($wqm list | sed 1d | wc -l)
+running_vm=$($wqm list | sed 1d | grep -c running )
+stopped_vm=$($wqm list | sed 1d | grep -c stopped )
+# Berechne LXCs
+total_lxc=$($wpct list | sed 1d | wc -l)
+running_lxc=$($wpct list | sed 1d | grep -c running )
+stopped_lxc=$($wpct list | sed 1d | grep -c stopped )
+# Berechne Gesamtwert
+total=$((total_vm + total_lxc))
+
+# wenn parameter = ...
+if [ "$1" = "autoconf" ]; then
```

Comment:
Please remove `autoconf`, if it is not used.

In this specific case I would recommend to verify the existence of `qm` and `pct`.

Reply:
changed it